Ubuntu/master.sh

```diff
@@ -166,6 +166,9 @@ while true
 cp /usr/bin/python3 /usr/bin/python
 apt install python3-pip -y 
 
+# UFW firewall you can deactivate it with this command
+sudo ufw disable
```

Comment:
```suggestion
ufw disable
```

Reply:
Since we run as root, sudo isn't needed. Also, is there a reason we need this disabled?